flags.go

```diff
@@ -48,6 +48,8 @@ var timeLimit = flag.Duration("duration", 0, "time to run for, 0 for infinite, e
 var meterID MeterIDFilter
 var meterType MeterTypeFilter
 
+var logAppend = flag.Bool("logappend", false, "apppend to log file if it exists")
```

Comment:
Please remove this flag, new log output handling flags are not something I want to deal with. At some point I'm going to remove all logging output related flags since console redirection should be sufficient.